CONTRIBUTING.md

```diff
@@ -19,3 +20,25 @@ find . -name "*.md" | while read -r file; do
     npx markdown-toc $file -i
 done
 ```
+
+# Using KubeLinter
+
+Please consider running [KubeLinter](https://docs.kubelinter.io/#/?id=usage)
+locally before submitting a PR to this repository.
+
+After [installing KubeLinter](https://docs.kubelinter.io/#/?id=installing-kubelinter)
+and adding it to the $PATH env variable, create a new folder in the base directory 
+using `mkdir -p ./.kube-linter/`. Then, run the following Bash script:
```

Comment:
@ShakedAviv50313 So eventually you weren't able to pipe the output of Kustomize to KubeLint ? Was there any technical barrier ?

Reply:
the steps given here that include creating a `./.kube-linter/` folder and then running the script are the exact same steps currently running on the CI

Reply:
But there's a difference between an ephemeral CI environment and running this locally on a non-ephemeral file system, right ?
Anyway, the process can probably be improved but I'm fine with the changes as they.